Modules/DataModel/Tests/PocketCastsDataModelTests/AutoAddCandidatesDataManagerTests.swift

```diff
@@ -0,0 +1,93 @@
+import XCTest
+import FMDB
+import PocketCastsUtils
+import SQLite3
+@testable import PocketCastsDataModel
+
+final class AutoAddCandidatesDataManagerTests: XCTestCase {
+
+    private var overriddenFlags = [FeatureFlag: Bool]()
+
+    private func override(flag: FeatureFlag, value: Bool) throws {
+        overriddenFlags[flag] = flag.enabled
+        try FeatureFlagOverrideStore().override(flag, withValue: value)
+    }
+
+    private func reset(flag: FeatureFlag) throws {
+        if let oldValue = overriddenFlags[flag] {
+            try FeatureFlagOverrideStore().override(flag, withValue: oldValue)
+        }
+    }
+
+    private func setupDatabase() throws -> DataManager {
+        let dbPath = (DataManager.pathToDbFolder() as NSString).appendingPathComponent("podcast_testDB.sqlite3")
```

Comment:
Do we need to expose `pathToDbFolder` to be used here? Can't we just use any other path?